### PR TITLE
Law: planetary total energy equation

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    v_r
+
+Latex:
+    :math:`v_r`
+"""
+
+potential_energy = Symbol("potential_energy", units.energy)
+"""
+..
+    FIXME Link document below.
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py:46]: What's this?

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py:46]: I can't seem to find a way how to reference another file from within the documentation. I've found there's a `:doc:` directive for that, but it gives an error.

Comment (blackyblack) [reply symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py]: I fixed it for you.

File: symplyphysics/laws/gravity/radial_motion/potential_energy_of_planetary_motion.py [hunk 2]
```diff
@@ -20,19 +20,25 @@
 )
 
 planetary_potential_energy = Symbol("planetary_potential_energy", units.energy)
-"""
+r"""
 The total potential energy of the planet.
 
 Symbol:
-    V
+    U_tot
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/radial_motion/potential_energy_of_planetary_motion.py:27]: Mark all symbols with `:code:`

File: symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/laws/gravity/radial_motion/total_energy_is_radial_kinetic_energy_plus_potential_energy.py]: Thank you, it wasn't obvious.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**